builder/bundles.go

```diff
@@ -833,7 +842,10 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	for _, e := range versions {
 		// TODO: change users of "versions" file to not rely on this exact formatting (version
 		// starting at column 51). E.g. this doesn't handle very well packages with large names.
-		fmt.Fprintf(w, "%-50s%s\n", e.name, e.version)
+		_, err = fmt.Fprintf(w, "%-50s%s\n", e.name, e.version)
```

Comment:
Interestingly, a `bufio.Writer` *can* return an error so you're doing the right thing by checking it here. That's interesting.